apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SessionPool.cs

```diff
@@ -38,7 +38,7 @@ namespace Google.Cloud.Spanner.V1
     /// and it will incur a penalty on other Spanner processes because it will be an hour before the server frees the session if 
     /// it's not properly deleted.
     /// </summary>
-    internal static class SessionPool
+    public static class SessionPool
```

Comment:
Happy to think about it - and if there's some way we could do this automatically, that would be great :) We do have the ability to hook modify requests shortly before they're sent, if that's useful. (There are partial methods generated.) When's the right time to investigate this?